### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 7

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#161] [Esercizi - Bertini] [answered]
![Screenshot 2023-10-23 185159](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/a5bcd659-6b15-4388-8614-4e030b9ce745)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/cb5bc33e-297a-440e-a173-446f29287af0)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Confermata in Classe
Sia $p=18/37$ la probabilità di vincere una partita e $1-p=19/37$ di perderla.
Le singole partite sono eventi indipendenti, le codifico con **0** in caso di perdita e **1** in caso di vincita.
Codifico la serie di 10 partite con uno schema di Bernoulli $XXXXXXXXXX$, dove $X=0,1$: tutti i possibili risultati delle **10** partite giocate sono $2^{10}$.

- **(1)** Armando effettua la prima vincita alla 5° partita se perde le prime 4, vince la 5° e consegue un qualsiasi risultato nelle rimanenti, ovvero $\mathbb{P}(00001XXXXX)=(1-p)^4\cdot p=(19/37)^4\cdot 18/37\simeq 0.034$
- **(2)** La prob che Armando vinca almeno 2 partite è la somma delle prob che Armando ne vinca **esattamente 2, esattamente 3, ..., esattamente 8** (eventi disgiunti):
$$\mathbb{P}(almeno\ due\ 1)=\sum^{10}\_{k=2}\binom{10}{k}p^k\cdot(1-p)^{10-k}=-[(1-p)^{10}+10p(1-p)^9]+\sum^{10}\_{k=0}\binom{10}{k}p^k\cdot(1-p)^{10-k}$$
Ora posso applicare la formula del Binomio di Newton, con $a=(1-p),\ b=p$:
$$-[(1-p)^{10}+10p(1-p)^9]+(1-p+p)^n=1-[\frac{19^{10}}{37^{10}}+\frac{10\cdot 18\cdot 19^9}{37\cdot 37^9}]\simeq 1- 0.23=0.77$$
Ovvero abbiamo calcolato l'evento complementare: la probabilità di vincerne **esattamente 0 o esattamente 1** (che potevamo fare dall'inizio :])
- **(3)** Il capitale di Armando aumenta di (esattamente) 2 euro se vince 6 partite e ne perde 4: 
$$\mathbb{P}(+2\ euro)=\mathbb{P}(sei\ 1)=\binom{10}{6}\cdot p^6\cdot (1-p)^4= 210\cdot(18/37)^6\cdot(19/37)^4\simeq 0.19$$

**Comment by cornflexxx** [comment]
$2)$ Nel secondo punto ho provato a passare a complemento ottenendo lo stesso risultato:

$$\mathbb{P}(\text{almeno due vinte})=1-\mathbb{P}(\text{esattamente una vinta $\cup$ esattamente zero vinte})=1-(\mathbb{P}(\text{esattamente una vinta}) \cup \mathbb{P}(\text{esattamente zero vinte}))$$

$$\mathbb{P}(\text{almeno due vinte})=1-[10(1-p)^9p+(1-p)^{10}]$$

**Comment by IonutCicio** [comment]
$1)$ Abbiamo $Z \sim \text{Geom}(\frac{18}{37})$

Per i punti $2)$ e $3)$, possiamo considerare $X \sim \text{Bin}(10, \frac{18}{37})$

$2)$ Abbiamo $\mathbb{P}(X >= 2) = 1 - \[\mathbb{P}(X = 0) + \mathbb{P}(X = 1)\]$ etc...

$3)$ Possiamo considerare $Y = 2 X - 10 \implies \mathbb{P}(Y = 2) = \mathbb{P}(2X - 10 = 2) = \mathbb{P}(X = 6) = {10 \choose 6} (\frac{18}{37})^6 (\frac{19}{37})^4$
